app/models/item.rb

```diff
@@ -0,0 +1,13 @@
+class Item < ApplicationRecord
+  belongs_to :merchant
+  validates_presence_of :name, :description, :price, :image, :inventory
+
+  def active?
+    if self.inventory > 0
+      self.active = true
+    else
+      self.active = false
+    end
+  end
```

Comment:
This is interesting. I'm assuming the reason why you have this is because you want to get a boolean back from a method that has a `?` at the end. You can name your columns with a question mark, and looking ahead at some of your migrations, it looks like you ended up doing that, so there is no need for this method. That being said, after all of your deletions and additions in your db, I'm still getting a column called `active` and a column called `active?`, I bet this may have made some of this confusing, so always check your schema to make sure you have the correct columns that you're anticipating your table to have.